docs/rst/tutorials/core/qos/mutable/mutable.rst

```diff
@@ -0,0 +1,267 @@
+In this tutorial, that same mechanism is used.
+In the private section of the `Node_ChangeMutableQoS_Publisher` class, the pointers to the native handlers are declared:
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/change_mutable_qos_publisher.cpp
+    :language: c++
+    :lines: 89-92
+    :dedent: 4
+
+
+In the constructor, the pointers are populated by calling the APIs provided by the rmw and rmw_fastrtps_cpp, until obtaining the `eprosima::fastdds::dds::DataWriter` handle:
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/change_mutable_qos_publisher.cpp
+    :language: c++
+    :lines: 52-55
+    :dedent: 4
+
+
+When the `Publisher_X_ownership_strength` is updated (for instance, via command line using `ros2 param set` command), the `cb` parameter callback is raised, and the `eprosima::fastdds::dds::DataWriter` handle is used to update its ownership strength.
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/change_mutable_qos_publisher.cpp
+    :language: c++
+    :lines: 64-78
+    :dedent: 4
+
+
+In this case, as in the current version of Fast DDS the builtin statistics are enabled by default (see `DomainParticipantQos <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/domain/domainParticipant/domainParticipant.html#domainparticipantqos>`_), it is needed to retrieve the internal QoS by means of `::get_qos()`, then perform the modifications and update the QoS by means of `::set_qos()`:
+The value of the ownership strength is set from the value of the updated parameter.
+
+Configuration of initial QoS
+----------------------------
+
+Ownership Strength Policy is mutable, but Ownership Policy is not. Then, it is needed to configure EXCLUSIVE_OWNERSHIP_POLICY to all participants before running the ROS nodes.
+To do that, inside our package, we have three xml files.
+Each one of them defines a profile for a publisher with a "large" ownership strength, another with a "small" ownership strength and a subscriber (that does not need an ownership strength definition).
+For the three of them, exclusive ownership is defined.
+
+.. tabs::
+
+    .. tab:: large_ownership_strength.xml
+
+        .. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/large_ownership_strength.xml
+            :language: xml
+
+    .. tab:: small_ownership_strength.xml
+
+        .. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/small_ownership_strength.xml
+            :language: xml
+
+    .. tab:: subscriber_exclusive_ownership.xml
+
+        .. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/subscriber_exclusive_ownership.xml
+            :language: xml
+
+
+Build
```

Comment:
We already built before. Maybe we can remove the other one and leave this one, up to you